house/forms.py

```diff
@@ -21,5 +22,18 @@ def __init__(self, *args, **kwargs):
         self.fields['children'].required = False
 
 
-class HouseIDForm(forms.Form):
-    house_id = forms.UUIDField(label='Enter Your House ID')
+class HouseCreationForm(forms.Form):
+    name = forms.CharField(label='House Name', required=True)
+    parent_profession_1 = forms.ChoiceField(choices=Job.choices, required=True)
+    parent_profession_2 = forms.ChoiceField(choices=Job.choices, required=True)
+    country = forms.ModelChoiceField(queryset=Country.objects.all(), required=True)
+    city = forms.ModelChoiceField(queryset=City.objects.all(), required=True)
+    income = forms.IntegerField(label='Monthly Income', required=True)
+    children = forms.IntegerField(label='Number of Children', required=True)
+    public = forms.BooleanField(label='public', required=True, initial=True)
```

Comment:
Like Shay commented in Rony's PR this shouldn't be required

Reply:
She needs to rebase and it will be fixed

Reply:
what Rony said :)